packages/grafana-ui/src/components/Dropdown/ButtonSelect.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { SelectableValue } from '@grafana/data';
+
+import { ButtonSelect } from './ButtonSelect';
+
+const OPTIONS: SelectableValue[] = [
+  {
+    label: 'Hello',
+    value: 'a',
+  },
+  {
+    label: 'World',
+    value: 'b',
+  },
+];
+
+describe('ButtonSelect', () => {
```

Comment:
I wrote theses tests to make sure that FocusScope wasn't entirely broken within jest, so they just came along for the ride 😅